tests/Unit/ConfigTest.php

```diff
@@ -55,3 +59,24 @@
         ])
         ->and($config->whitelistedPaths)->toBe([]);
 });
+
+it('can set ignore words', function (): void {
+    if (file_exists($filePath = dirname(__DIR__).'/peck-testing.json')) {
+        unlink($filePath);
+    }
+    Config::flush();
+    Config::resolveConfigFilePathUsing(
+        fn (): string => $filePath,
+    );
+
+    Config::init();
+    $config = Config::instance();
+
+    expect($config->whitelistedWords)->toBe(['php']);
+
+    $config->ignoreWords(['laravel', 'paravel']);
+    Config::flush();
+    unlink($filePath);
+
+    expect($config->whitelistedWords)->toBe(['php', 'laravel', 'paravel']);
```

Comment:
Here we can also assert that the persist changed the file and check if the file contains the words as well.